circle.yml

```diff
@@ -14,6 +14,7 @@ dependencies:
         - brew update
         - brew tap homebrew/homebrew-php
         - brew install php71
+        - echo "memory_limit = 512M" > /opt/circleci/php/$(phpenv global)/etc/conf.d/memory.ini
```

Comment:
Since it uses brew, why not try:

```bash
mkdir $(brew --prefix)/etc/php/7.1/conf.d
echo "memory_limit = 512M" > $(brew --prefix)/etc/php/7.1/conf.d/memory.ini
```

Reply:
my change doesn't work, and TBH I've no idea what I'm doing here ;)
lemme try your suggestion